src/views/view-identity-attributes.njk

```diff
@@ -0,0 +1,63 @@
+{% extends "layouts/govuk_template.html" %}
+
+{% block head %}
+<link href="css/app.css" media="screen" rel="stylesheet">
+{% endblock %}
+
+{% block content %}
+<main id="content">
+<h2 class="heading-medium">Attributes for user with pid: {{identity.pid}} (LOA: {{identity.levelOfAssurance}})</h2>
+<p>
+<table>
+    <thead>
+        <tr>
+            <th>Attribute Name</th><th>Attribute Value</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>First Name</td>
+            <td><code>{{identity.attributes.firstName.value}}</code></td>
+        </tr>
+        {% for attribute in identity.attributes.middlenames %}
+          <tr>
+              <td><code>Middle Name</td>
+              <td><code>{{attribute.value}}</code></td>
```

Comment:
Could we add whether the attribute is `verified` here?

Reply:
done